sorc/global_cycle.fd/cycle.f90

```diff
@@ -42,11 +42,14 @@
 !!                     file.
 !!  - $NST_FILE        Gaussian GSI file which contains NSST
 !!                     TREF increments
-!!  - $LND_SOI_FILE.$NNN    Gaussian GSI file which contains soil state
+!!  - $GSI_SOI_FILE.$NNN    Gaussian GSI file which contains soil state
 !!                     increments
-!!  - xainc.$NNN       The cubed-sphere increment file (contains 
+!!  - xainc.$NNN       The cubed-sphere snow increment file (contains 
```

Comment:
Change to "snow_xainc" throughout, for consistency with the soil option.

Reply:
Done.